Assets/HoloToolkit/Utilities/Scripts/DataEventArgs.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+
+namespace HoloToolkit.Unity
+{
+    public static class DataEventArgs
+    {
+        public static DataEventArgs<TData> Create<TData>(TData data)
+        {
+            return new DataEventArgs<TData>(data);
+        }
+    }
+
+    [Serializable]
+    public class DataEventArgs<TData> :
+        EventArgs
```

Comment:
I think breaking on methods with long parameters is okay, but I think it's more for consistency.